environments/202402/0-desy-pinned.yml

```diff
@@ -3,35 +3,30 @@ channels:
   - conda-forge
   - nodefaults
 dependencies:
-  - brotlipy=0.7.0=py39hb9d737c_1004
   - certipy=0.1.3=py_0
-  - ipykernel=6.13.1=py39hef51801_0
-  - ipympl=0.9.3=pyhd8ed1ab_0
-  - ipython=7.33.0=py39hf3d152e_0
-  - ipython_genutils=0.2.0=py_1
-  - ipywidgets=7.7.0=pyhd8ed1ab_0
-  - jupyter=1.0.0=py39hf3d152e_7
-  - jupyter-resource-usage=0.7.1=pyhd8ed1ab_0
+  - ipykernel=6.29.0=pyhd33586a_0
+  - ipympl=0.9.4=pyhd8ed1ab_0
```

Comment:
Can we update the docs to mention these versions? https://european-xfel.github.io/environments/pr/49/environments/#interactive-plotting-issues-in-jupyter-notebooks

Reply:
I sneakily updated it in b89c989 :face_in_clouds: